src/pages/deposit.vue

```diff
@@ -390,6 +393,9 @@ export default Vue.extend({
         this.transactionInfo.fee = undefined;
         this.tokenAllowance = await this.getTokenAllowance(this.chosenToken);
         this.transactionInfo.continueBtnFunction = true;
+        if (this.amountBigNumber.gt(this.tokenAllowance)) {
+          this.inputtedAmount = utils.handleFormatToken(this.chosenToken.symbol, this.tokenAllowance);
+        }
         if (!unlimited) {
```

Comment:
Replace it with ternary operand like:
```this.transactionInfo.continueBtnText = unlimited ?  "Proceed to deposit" : "Ok"```
BTW, isn't "Proceed to deposit" the result of the unlimited allowance?

Reply:
Nope